rocketmq-spring-boot/src/main/java/org/apache/rocketmq/spring/autoconfigure/RocketMQAutoConfiguration.java

```diff
@@ -94,9 +95,10 @@ public RocketMQTemplate rocketMQTemplate(DefaultMQProducer mqProducer, ObjectMap
     }
 
     @Bean
-    @ConditionalOnBean(RocketMQTemplate.class)
+    @ConditionalOnBean(name = RocketMQConfigUtils.ROCKETMQ_TEMPLATE_DEFAULT_GLOBAL_NAME)
     @ConditionalOnMissingBean(TransactionHandlerRegistry.class)
-    public TransactionHandlerRegistry transactionHandlerRegistry(RocketMQTemplate template) {
+    public TransactionHandlerRegistry transactionHandlerRegistry(@Qualifier(RocketMQConfigUtils.ROCKETMQ_TEMPLATE_DEFAULT_GLOBAL_NAME)
```

Comment:
Another problem:

Transactional Message used with default RocketMQTemplate.  Can it also be used with multiple RocketMQTemplate.

Reply:
Transactional Message  still uses the default rocketMQTemplate in the implement.